content/yaml-quick-start/white-label-getting-started.md

```diff
@@ -61,9 +61,9 @@ To add an environment variable using the Codemagic REST API, you need your API a
 
 - The access token is available in the Codemagic UI under **Teams > Personal Account > Integrations > Codemagic API > Show**. You can then store this as an environment variable if you are calling the REST API from other workflows.
 - Once you have added your app in Codemagic, open its settings and copy the **application id** from the browser address bar - `https://codemagic.io/app/<APP_ID>/settings`
-- The client group name, is the group that holds all variables for this client e.g. `WL_001`, `WL_002`.
+- The client group name is the group that holds all variables for this client e.g. `WL_001`, `WL_002`.
 
-An example of adding a secure variable to an application group looks like this:
+An example of adding a secret variable to an application group looks like this:
 
 {{< highlight bash "style=paraiso-dark">}}
```

Comment:
TODO: update API example

Reply:
Will leave the API example update out of scope as something to be reviewed separately.